### DISCUSSION TITLE
[Error] GetCollection failed: Unable to complete request, Internal Server Error

### DISCUSSION THREAD

**Discussion by SilverDrg** [tinacms/tinacms#3593] [Help]
After updating a few posts yesterday I haven't been able to access them on the `/admin` panel on my website, each time I receive an Internal server error when trying to fetch the collections. 
Additionally on Tina Cloud when trying to reindex any of the branches it results in a `NetworkError when attempting to fetch resource.` with `no branches found with data layer enabled`  being the response in the network tab.
I have also tried refreshing webhooks which also results in a network error with a forbidden response.

The posts are still able to be fetched to display on my website but not when trying to fetch them in the `/admin` dashboard

I'm a bit lost as to what the issue could be, if anyone has any idea please let me know.
Any help is appreciated, thanks.

**Comment by logan-anderson** [comment]
@SilverDrg What version of TInaCMS are you using? We made an update recently that might have caused an issue for older versions.

If you are on an older version updating is probably the best fix. You can find different update guides for the different versions in our blog section of our site.

The largest change is Probably here: https://tina.io/blog/upgrading-to-iframe/

**Reply by SilverDrg** [reply]
This might be the case, I am currently on version 0.68. I will try updating it and see if that fixes the issue

**Reply by Viktorija97** [reply]
Hi, what fixed my problem for a project that is on 0.60 version is adding this line --experimentalData in tinacms server:start  -c \"next dev\""
